docker/openchallenges/services/mcp-server-standalone.yml

```diff
@@ -0,0 +1,14 @@
+services:
```

Comment:
(Optional) It would be nice to include a comment on top that briefly describe this MCP server and how to start it using this file (`docker compose ...`).